src/ontology/edits/oeo-physical.omn

```diff
@@ -1302,7 +1305,7 @@ Class: HydrogenTurbine
     
     SubClassOf: 
         Turbine,
-        has_physical_output some Power,
+        has_physical_output some PowerValue,
         has_physical_input only Hydrogen
```

Comment:
For logical consistency: If a transformer has a value as output then it also needs an value as input.